server/utils/latest.sql

```diff
@@ -0,0 +1,43 @@
+DELETE FROM pages;
```

Comment:
I think you can drop the table.
drop table pages;

Also, the order is important.
you cannot drop pages (or even delete pages) before deleting modules because there is a reference.